entities/automation/sensor/tomorrow_io_weather_code/rain_with_open_door.yaml

```diff
@@ -0,0 +1,35 @@
+---
+alias: /sensor/tomorrow-io-weather-code/rain-with-open-door
+id: sensor_tomorrow_io_weather_code_rain_with_open_door
+description: Alert Will when it starts to rain and the roof terrace door is open
+
+mode: single
+
+trigger:
+  - platform: state
+    entity_id: sensor.tomorrow_io_weather_code
+    to:
+      - "Drizzle"
+      - "Rain"
+      - "Light Rain"
+      - "Heavy Rain"
+      - "Freezing Drizzle"
+      - "Freezing Rain"
+      - "Light Freezing Rain"
+      - "Heavy Freezing Rain"
+      - "Thunderstorm"
```

Comment:
Use the rain intensity sensor, and also set it so that if it goes back to 0 the notification is cleared @cursoragent